### PR TITLE
template database handling

### PR DESCRIPTION
**Pull request by tarzan0820**
*Add a menuitem via which technical users can view template databases
*When "Delete Template" is clicked there is no reason to still keep the saas_portal.database

### CODE REVIEW

File: saas_portal/models/saas_portal.py [hunk 1]
```diff
@@ -306,7 +306,9 @@ def upgrade_template(self):
 
     @api.multi
     def delete_template(self):
-        return self[0].template_id.delete_database()
+        res = self[0].template_id.delete_database()
+        self.unlink()
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: I don't agree with you here. A user can delete template to create it from scratch again. E.g. if something went wrong with template (he update some custom modules, and it broke odoo database)/

**Comment by yelizariev** [comment]
THank you for saas_portal.database views. 

Check my opinion about deleting plan when template is deleted

**Comment by saltonmassally** [comment]
@yelizariev I have removed the link that deletes

**Comment by yelizariev** [comment]
thank you

**Pull request merged**